COPYING

```diff
@@ -0,0 +1,15 @@
+Copyright 2022 Stellar Development Foundation and contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+As well as the file LICENSE adjacent to this file.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
I don't think we need to follow the GNU pattern of including a COPYING file. The LICENSE file is sufficient. Can we drop this? Or do we need it for some specific reason. Fwiw we don't include a COPYING in most repos.

Reply:
You have one in the go repo, we have one in core .. I'm not sure what you mean by "most"

Reply:
(The LICENSE file does not say who owns the copyright or connect that ownership to an act of licensing -- it's a different statement)

Reply:
According to GitHub we don't include it in most of our repos, but you're right, does look like it is in go too. Sounds fine to me, just seems unnecessary.

Reply:
https://github.com/stellar/rs-stellar-contract-env/blob/eafb7d5a8b17886844b059724897af3803403b72/LICENSE#L190-L196